drivers/UnbufferedSerial.h

```diff
@@ -23,6 +23,7 @@
 
 #include <cstdarg>
 
+#include "platform/Stream.h"
```

Comment:
We don't want to include Stream.h. Why did you add it?

Reply:
There was build failure on missing Stream.h header and I noticed that this file included previously via RawSerial.h file so I included this into UnbufferedSerial.h

Reply:
The build failure should be solved in a different way. `UnbufferedSerial` does not need `Stream`.